daemon/lokinet.cpp

```diff
@@ -419,6 +419,11 @@ namespace
         }
       }
 
+      if (options.router)
```

Comment:
why not `opts.isSNode = options.router;` ?

Reply:
resolved and merged into previous commit